src/Behat/Config/Converter/UseStatementAdder.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Behat\Config\Converter;
+
+use PhpParser\Node;
+use PhpParser\NodeVisitorAbstract;
+
+class UseStatementAdder extends NodeVisitorAbstract
+{
+    /**
+     * @var array <string, class-string>
+     */
+    private array $uses = [];
+
+    public function enterNode(Node $node): ?Node
+    {
+        if ($node instanceof Node\Name\FullyQualified) {
+            $className = $node->toString();
+            $shortName = $node->getLast();
+
+            $this->uses[$shortName] = $className;
+
+            return new Node\Name($shortName);
+        }
+        return null;
```

Comment:
Perhaps use an early return here?

```suggestion
        if (!$node instanceof Node\Name\FullyQualified) {
            return null;
        }
        
        $className = $node->toString();
        $shortName = $node->getLast();

        $this->uses[$shortName] = $className;

         return new Node\Name($shortName);
```

Reply:
Updated